yarn.lock

```diff
@@ -3773,17 +3809,17 @@ __metadata:
     zod: "npm:^3.21.4"
   peerDependencies:
     ethers: ">=5.6.0 < 6.0.0"
-  checksum: 91340d7b545ef031d95faa5fbbd87f0056d13ebb08e51d7ad7b63035e615e2a735d1703c07b667ff5d6111c64e84da8f531063de6c4d7ccd8cb64f332f987613
+  checksum: 9f2eca64e386ccb43c7e8d693d9b3eafc66b7bc599367e23085dc7b1747d8d52c6d84740cafddadc3ef8143512908a5feb0aa71be6be5fd68b5fd349d6088b58
   languageName: node
   linkType: hard
 
-"@usecannon/cli@npm:^2.9.3":
-  version: 2.9.3
-  resolution: "@usecannon/cli@npm:2.9.3"
+"@usecannon/cli@git+https://github.com/usecannon/cannon.git#commit=be8939728ebb1ad26f835c83607d8d2973300153&workspace=@usecannon/cli":
+  version: 2.9.8
```

Comment:
Somehow not all updated and here we have 2 cannon builders. 

Use resolutions in package.json instead of updating each package.json